src/components/Signals/Signals.tsx

```diff
@@ -0,0 +1,79 @@
+import { useEffect, useState } from 'react'
+
+import clsx from 'clsx'
+
+import useSignals from './hooks'
+
+import styles from './Signals.module.scss'
+
+type ActiveEndpointRes = {
+  active: boolean
+  language: 'en' | 'no' | 'fi' | 'da'
+  widgetUrl: string
+  questionType: 'calendarStep' | 'checkboxes' | 'multipleChoice' | 'studyPanel'
+  header: string
+  description: string
+  options: Array<{
+    label: string
+    position: number
+    answerUrl: string
+  }>
+}
+
+interface Props {
+  id: string
+  width?: string
+  breakpoint?: 'xs' | 'sm' | 'md' | 'lg' | 'xl' | 'xxl'
+  demo?: boolean
+}
+
+export const Signals = ({
+  id,
+  width = '632px',
+  breakpoint = 'xs',
+  demo = false,
+}: Props) => {
+  const [isActive, setIsActive] = useState(demo || false)
```

Comment:
Denne trenger jo egentlig ingen fallback
```suggestion
  const [isActive, setIsActive] = useState(demo)
```